lib/Auth/Utils.php

```diff
@@ -21,7 +21,7 @@ class Utils
      * @return PIResponse|null An array containing attributes and detail, or NULL.
      * @throws Exception
      */
-    public static function authenticatePI(array &$state, array $formParams): ?PIResponse
+    public static function authenticatePI(array &$state, array $formParams, array $headersToForward): ?PIResponse
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)